client/client.go

```diff
@@ -104,6 +104,17 @@ type (
 		//	- InternalServiceError
 		SignalWorkflow(ctx context.Context, workflowID string, runID string, signalName string, arg interface{}) error
 
+		// SignalWithStartWorkflow sends a signals to a workflow in execution, if workflow is not running or not found,
+		// it starts workflow then sends the signal in transaction.
+		// - workflowID, runID, signalName, signalArg are same as SignalWorkflow's parameters
+		// - options, workflow, workflowArgs are same as StartWorkflow's parameters
+		// The errors it can return:
+		//  - EntityNotExistsError, if domain does not exists
```

Comment:
suggested update:
SignalWithStartWorkflow can return the following errors:
- EntityNotExistsError, if the domain does not exist